language-design.md

```diff
@@ -28,6 +83,14 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
+One noticeable characteristic of a well-designed langauge is that it makes space
+for growth. User are able to grow the language themselves, following the bazaar
+metaphor in the reading from Steele. Another similar idea is that users are
+able to build things in the language that looks like things that are built into
+the language to give them a sense ownership. If the language is inflexible, it
+makes it hard for the user to make things. Furthermore, if it's built large and
+without collaboration, it is likely designed in a way that is not what the users
+would like.
```

Comment:
All very valid points. Just to play the devil's advocate, what about Java as a programming language? It's not one where users have had much say in its development, yet it is widely used and seems to be well-designed (in my understanding, at least). Why do you think it's been quite successful despite not following the bazaar metaphor?